packages/lib-sourcify/src/lib/SourcifyChain.ts

```diff
@@ -239,4 +323,45 @@ export default class SourcifyChain {
         this.chainId
     );
   };
+
+  getContractCreationBytecode = async (
+    address: string,
+    transactionHash: string
+  ): Promise<string> => {
+    const txRecipt = await this.getTxRecipt(transactionHash);
+    const tx = await this.getTx(transactionHash);
+    let creationBytecode = '';
+
+    if (txRecipt.contractAddress !== null) {
```

Comment:
Is this the address of the contract created? Does all RPCs return this? 

I thought this value wasn't always returned. That's why we double check the contract address to be created in `matchWithCreaetionTx` using EOA address + nonce

Reply:
exactly, read the lines later:

```typescript
if (txReceipt.contractAddress !== null) {
      // EOA created
} else {
      // Factory created
```